okhttp/src/jvmMain/kotlin/okhttp3/Cookie.kt

```diff
@@ -263,6 +283,7 @@ class Cookie private constructor(
     private var httpOnly = false
     private var persistent = false
     private var hostOnly = false
+    private var sameSite = "lax"
```

Comment:
This is documented as `Lax` with a capital `L` ??

Reply:
+1 - seems like we should prefer the typical case.

Reply:
I believe the default should be `"None"`